### DISCUSSION TITLE
Newest AWS lambda runtime fails to decrypt

### DISCUSSION THREAD

**Discussion by kkayyali-magnet** [openpgpjs/openpgpjs#1753] [Q&A]
- OpenPGP.js version: [5.0.0-3](https://www.npmjs.com/package/openpgp/v/5.0.0-3)
- Affected platform (Browser or Node.js version): AWS Lambda "Node18.x "v18.28 RuntimeARN: arn:aws:lambda:us-east-1::runtime:b475b23763329123d9e6f79f51886d0e1054f727f5b90ec945fcb2a3ec09afdd

Works fine on previous runtime (ARN: arn:aws:lambda:us-east-1::runtime:464db84005d4f68e67934a6df703e04e6e8782a50ec2882673a8e1c996ab814a) Node 18.21

Identical code base

Only error is

`"stack": "Error: Error decrypting message: Decryption error\n    at /opt/nodejs/node_modules/openpgp/dist/node/openpgp.min.js:2:178290\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async go.decrypt (/opt/nodejs/node_modules/openpgp/dist/node/openpgp.min.js:2:283085)\n    at async /opt/nodejs/node_modules/openpgp/dist/node/openpgp.min.js:2:335316\n    at async Promise.all (index 0)\n    at async /opt/nodejs/node_modules/openpgp/dist/node/openpgp.min.js:2:335178\n    at async Promise.all (index 0)\n    at async /opt/nodejs/node_modules/openpgp/dist/node/openpgp.min.js:2:334785\n    at async Promise.all (index 0)\n    at async oc.decryptSessionKeys (/opt/nodejs/node_modules/openpgp/dist/node/openpgp.min.js:2:334742)",`

Wondering if there is a change to a dep, or perhaps crypto changed enough to cause an issue.

**Comment by larabr** [comment]
You are on a pre-release version of the library.
Please upgrade to the latest version and see if the problem persists ([v5.11.1](https://github.com/openpgpjs/openpgpjs/releases/tag/v5.11.1)).
